### PR TITLE
Fix missing translations

### LINKED ISSUE #1495: Release 4.0
I'm using this branch to test the upgrade from 3.x branch to 4.x

# Those issues need to be solved before release 4.0 version

CSS / Javascript:

Other:
- https://github.com/symfony/recipes-contrib/pull/1440 update recipe 
- [x] https://github.com/sonata-project/SonataBlockBundle/pull/1124 Menu block is returning an internal server error: https://github.com/sonata-project/SonataPageBundle/issues/1495#issuecomment-1269493681

Completed:
- [x] #1627
- [x] https://github.com/sonata-project/SonataSeoBundle/pull/696
- [x] #1624
- [x] https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1695 , and `requires ^4.8`, Batch delete for shared blocks does not work: https://github.com/sonata-project/SonataPageBundle/issues/1495#issuecomment-1252027436
- [x] #1623 Composer page is not working when there are blocks that doesn't exist
- [x] #1609
- [x] #1474
- [x] #1615 List label for snapshot list was not found: https://github.com/sonata-project/SonataPageBundle/issues/1495#issuecomment-1252032137
- [x] https://github.com/sonata-project/SonataAdminBundle/pull/7922 #1615 Composer page is not showing errors: https://github.com/sonata-project/SonataPageBundle/issues/1495#issuecomment-1235626107
- [x] #1618 I can't select a shared block into my page: https://github.com/sonata-project/SonataPageBundle/issues/1495#issuecomment-1252021888
- [x] Remove deprecate blocks from [base_layout.html.twig](https://github.com/sonata-project/SonataPageBundle/blob/4.x/src/Resources/views/base_layout.html.twig#L23)
- [x] #1586
- [x] #1583 #1573
- [x] Test if multi site works with the current code!
- [x] #1596 Deprecate css/js configuration in 3.x and remove in 4.x
- [x] The block link is not going to edit form, look this comment: https://github.com/sonata-project/SonataPageBundle/issues/1495#issuecomment-1226906394
- [x] Block show is empty, Look this comment: https://github.com/sonata-project/SonataPageBundle/issues/1495#issuecomment-1226906394
- [x] #1581 Add an upgrade note to remove `sonata_block.context_manager: sonata.page.block.context_manager` and `sonata_page.slugify_service: sonata.page.slugify.cocur`
- [x] #1497 
- [x] #1579 Composer page is not adding block types
- [x] Composer page is not working: https://github.com/sonata-project/SonataPageBundle/issues/1495#issuecomment-1211637698
- [x] #1496 https://github.com/sonata-project/SonataPageBundle/pull/1532
- [x] #1605 Check the navigation links, in snapshots, site and blocks admin
- [x] Make sure that blocks removed on 4.x was deprecated in 3.x branch, to avoid errors like this `An exception has been thrown during the rendering of a template ("The block service `sonata.seo.block.email.share_button` does not exist").`
- [x] #1498 
- [x]  #1502 `http://localhost:8001/admin/app/sonatapagesite/list`  is returning this internal server error: `An exception has been thrown during the rendering of a template ("Cannot access property "create_snapshots" in class "App\Entity\SonataPageSite".").`
- [x] #1505 Create page is missing some translations: `form.group_form_page_group_main_label`, `form.group_form_page_group_seo_label`, `form.group_form_page_group_advanced_label` look here: https://github.com/sonata-project/SonataPageBundle/issues/1495#issuecomment-1195645922
- [x] http://localhost:8001/admin/app/sonatapagepage/tree is returning `Action "tree" could not be found in access mapping. Please make sure your action is defined into your admin class accessMapping property.`
- [x] #1524 at `http://localhost:8001/admin/dashboard` is returning `An exception has been thrown during the rendering of a template ("The options "manager", "page_id" do not exist. Defined options are: "attr", "extra_cache_keys", "groups", "template", "ttl", "use_cache".").`
- [x] when I'm running `composer update -W`, I'm facing this issue `Compile Error: Type of App\Entity\SonataPageBlock::$page must be ?Sonata\Pa  
!!    geBundle\Model\PageInterface (as in class Sonata\PageBundle\Model\Block) ` at `In SonataPageBlock.php line 16:`
- [x] When I access the composer page (`http://localhost:8001/admin/app/sonatapagepage/642/compose`), I got this error `Variable "admin_pool" does not exist.` https://github.com/sonata-project/SonataPageBundle/pull/1533
- [x] #1539 Snapshot edit page is missing group translation: https://github.com/sonata-project/SonataPageBundle/issues/1495#issuecomment-1210275144
- [x] #1535
- [x] Problem when going to create a new publication from `/admin/app/sonatapagesnapshot/create`: https://github.com/sonata-project/SonataPageBundle/issues/1495#issuecomment-1210757295
- [x] #1548 Create snapshot command is not working

### PR DESCRIPTION
**Pull request by Amin-Hosseini**
## Subject

I am targeting this branch, because fix missing some translations.

### CODE REVIEW

**Comment by eerison** [comment]
> IMO we can also remove some duplicate translate sources, for example form.group_general in xliff files.

well if you have duplicate translations, I guess you can remove and let only one

what do you think @VincentLanglet ?

**Comment by VincentLanglet** [comment]
Indeed if they are not used

File: src/Resources/translations/SonataPageBundle.bg.xliff [hunk 1]
```diff
@@ -522,8 +522,8 @@
         <source>form.label_locale</source>
         <target>Locale</target>
       </trans-unit>
-      <trans-unit id="131" resname="form_site.label_seo">
-        <source>form_site.label_seo</source>
+      <trans-unit id="131" resname="form.group_seo">
```
Comment (VincentLanglet) [inline src/Resources/translations/SonataPageBundle.bg.xliff]: Duplicate of id 35

File: src/Resources/translations/SonataPageBundle.bg.xliff [hunk 2]
```diff
@@ -562,8 +562,8 @@
         <source>list.label_edited</source>
         <target>Редактирана</target>
       </trans-unit>
-      <trans-unit id="141" resname="show.label_site">
-        <source>show.label_site</source>
+      <trans-unit id="141" resname="show.group_site">
```
Comment (VincentLanglet) [inline src/Resources/translations/SonataPageBundle.bg.xliff:561]: When it's used ? I dont see the related change in the code

File: src/Resources/translations/SonataPageBundle.bg.xliff [hunk 3]
```diff
@@ -615,12 +615,12 @@
         <source>title_select_block_type</source>
         <target>Избери блок тип</target>
       </trans-unit>
-      <trans-unit id="153" resname="form.field_group_general">
-        <source>form.field_group_general</source>
+      <trans-unit id="153" resname="form.group_general">
```
Comment (VincentLanglet) [inline src/Resources/translations/SonataPageBundle.bg.xliff]: Duplicate of 137

File: src/Resources/translations/SonataPageBundle.bg.xliff (hunk 2 above)
Comment (Amin-Hosseini) [reply src/Resources/translations/SonataPageBundle.bg.xliff:561]: Show site page: in `http://localhost:8001/admin/app/sonatapagesite/1/show`
I guess generator of translation keys changed from `show.label_site` to `show.group_site`

Comment (VincentLanglet) [reply src/Resources/translations/SonataPageBundle.bg.xliff:561]: Weird...

But if it works better this way, ok ! Thanks

Comment (Amin-Hosseini) [reply src/Resources/translations/SonataPageBundle.bg.xliff:565]: Page :
Screenshot 2022-07-28 at 16 56 29

Translation :
Screenshot 2022-07-28 at 16 55 10

**Review by VincentLanglet — APPROVED** [review]

**Comment by VincentLanglet** [comment]
No need for changelog since it's not release yet :)

**Review by jordisala1991 — APPROVED** [review]

**Comment by VincentLanglet** [comment]
Thanks

**Pull request merged**